intera_core_msgs/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ add_message_files( DIRECTORY msg
         IOStatus.msg
         IONodeStatus.msg
         URDFConfiguration.msg
+        InteractionControlCommand.msg
```

Comment:
Note for 5.2 - would these `InteractionControl` messages make more sense in `*_motion_msgs`? Fine (and required) that they be here for now.